multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ProcessDescriptorStep.java

```diff
@@ -96,6 +100,14 @@ private MtaDescriptorPropertiesResolverContext buildMtaDescriptorPropertiesResol
                                                               .build();
     }
 
+    private void setTimeoutGlobalParameters(ProcessContext context, DeploymentDescriptor descriptor) {
+        TimeoutGlobalParameters timeoutGlobalParameters = new TimeoutGlobalParameters(descriptor);
```

Comment:
why don't we set the default values at the 'Process'Metadata class level but do it here? Also - where do we set the values set at the descriptor level and the ones set at the command line level?